packages/filebrowser/src/listing.ts

```diff
@@ -922,7 +922,7 @@ export class DirListing extends Widget {
     this._onItemOpened.emit(item);
     if (item.type === 'directory') {
       this._model
-        .cd(item.path)
+        .cd(`/${item.path}`)
```

Comment:
Does it make sense to ensure the leading slash here?

```typescript
.cd('/' + PathExt.normalize(item.path))
```

Reply:
`item.path` documentation says it never has a leading slash: https://github.com/jupyterlab/jupyterlab/blob/e7c8c9ac5eb6a2c22cf5b3070a2926e8b0f44b62/packages/services/src/contents/index.ts#L50